TwitterCore/TwitterCore/Identity/TWTRAppleSocialAuthenticaticationProvider.m

```diff
@@ -207,46 +207,44 @@ - (void)showActionSheetWithCompletion:(TWTRAuthenticationProviderCompletion)comp
     TWTRParameterAssertOrReturn(completion);
 
     dispatch_async(dispatch_get_main_queue(), ^{
-        UIActionSheet *sheet = [self actionSheet];
-        objc_setAssociatedObject(sheet, (__bridge const void *)(TWTRSocialAppProviderActionSheetCompletionKey), completion, OBJC_ASSOCIATION_COPY_NONATOMIC);
-
-        // This shows the picker in the middle of the screen. It doesn't look good,
-        // and the UIAlertController on iOS 8 would be the solution for this, but we
-        // don't officially support iPad at this time so this will do.
-        [sheet showInView:[TWTRUtils topViewController].view];
+        UIAlertController *alertController = [self alertController];
+        objc_setAssociatedObject(alertController, (__bridge const void *)(TWTRSocialAppProviderActionSheetCompletionKey), completion, OBJC_ASSOCIATION_COPY_NONATOMIC);
```

Comment:
There is no need for using associatedObject as completion now could be passed to the UIAlertAction handlers when creating the alertController.

Reply:
Fixed in 2c24fa9